build.js

```diff
@@ -0,0 +1,35 @@
+const webpack = require('webpack'); // eslint-disable-line import/no-extraneous-dependencies
+const config = require('./webpack/webpack.prod.config.js');
```

Comment:
Is `build.js` still a prod specific file because it imports `webpack.prod.config.js`, but isn't named like it is a prod specific file?  Or, is it conditionally prod-like?
Can you either add a comment here, or update the name of this file, as appropriate depending on the answer?

Reply:
Good call.